frontends/infinite-corridor/src/api/learning-resources/hooks.test.tsx

```diff
@@ -198,7 +164,9 @@ test.each([
     expect(spies.queryClient.invalidateQueries).toHaveBeenCalledTimes(2)
 
     // The POST response result updated resource data
-    expect(resourceResult.current.data).toEqual(modifiedAddedResource)
+    await waitFor(() => {
+      expect(resourceResult.current.data).toEqual(modifiedAddedResource)
+    })
   }
```

Comment:
This line was checking that the POST result from `useAddToListItems` was used to update the List resource.

Previously, the update was happening syncrhonously after the POST request resolved. In v4, it seems to happen asyncrhonously (but very fast) after the request resolves.